### PR TITLE
Added tests for degree model

### PR DESCRIPTION
**Pull request by DanielItkes**
The tests are not working when running the pytest command in the VM, hoping to get some feedback as to why.

### CODE REVIEW

File: AcadeMeData/tests.py [hunk 1]
```diff
@@ -1,3 +1,13 @@
...
+import pytest
+from AcadeMeData.models import Degree
 
-# Create your tests here.
+
+@pytest
+class TestDegreeModel(pytest):
+    def test_get_name(self):
+        test_degree = Degree(degree='Computer Science').save()
+        assert test_degree.get_name() == 'Reichman University'
```
Comment (DanielItkes) [inline AcadeMeData/tests.py:9]: Should return false, as th name of the degree is 'Computer Science'.

File: AcadeMeData/tests.py [hunk 2]
```diff
@@ -1,3 +1,13 @@
...
+
+    def test_universities(self):
+        test_degree = Degree(degree='Economics', universities='Reichman University').save()
+        assert "Reichman University" in test_degree.universities()
```
Comment (DanielItkes) [inline AcadeMeData/tests.py:13]: Should return true, as 'Reichmain University' is in the string 'Reichman University'.

**Review by Yarboa — CHANGES_REQUESTED** [review]
It seems to me as part of PR #52 
Please add the test in that PR, thanks

**Comment by DanielItkes** [comment]
Hi Yariv it's not part of PR #52, the tests are not working and I've used this PR to get feedback
#52 is crucial for pushing a PR containing Course model (which is already done), as there is a dependancy on Degree model
The tests are not working for me on either one of them (Degree/Course models) because of an error I've documented on Slack

**Review by Yarboa — COMMENTED** [review]
Please lets continue in PR #52 
The tewst should enter there, since the tests are checking the Model Degree

**Comment by DanielItkes** [comment]
Okay I'll look into it tommorow, thanks!

**Comment by DanielItkes** [comment]
Outdated.

**Pull request closed without merging**